components/quest-boost/boostCard.tsx

```diff
@@ -14,121 +14,122 @@ import { TOKEN_DECIMAL_MAP } from "@constants/common";
 import { getTokenName } from "@utils/tokenService";
 
 type BoostCardProps = {
-  boost: Boost;
-  completedQuests?: number[];
+	boost: Boost;
```

Comment:
too many linting changes which don't comply with the project's linting. Can you please fix the lint config on your editor?